html/wifi/wifi.html

```diff
@@ -30,8 +30,7 @@
             <div id="aps">Scanning... <div class="spinner spinner-small"></div></div>
             <label for="opt-hiddenssid">
               <input type="radio" name="essid" value="_hidden_ssid_" id="opt-hiddenssid">
-              <input type="text" id="hidden-ssid" value=""
-                style="width:auto; display:inline-block; margin-left: 0.7em">
+              <input type="text" id="hidden-ssid" value="" style="width:auto; display:inline-block; margin-left: 0.7em">
             </label>
             <label>WiFi password, if applicable:</label>
```

Comment:
I just fixed the source indentation but I see the "WiFI" on line 12 and "Wifi" on line 13 now that you pointed it out.  Looks like a mixed bag of "WiFI" and "Wifi" throughout so I'l normalize everything to WiFI.